ezb_api/ezb_api.py

```diff
@@ -0,0 +1,342 @@
+from ezb_api_exceptions import RemoteCommunicationException
+import numpy as np
+import requests
+from xml.etree import ElementTree

```

Comment:
Should use some different library. The ElementTree loads the whole data to the RAM. We will run out of memory if we get way too much data. Probably should prefer something that will load data chunk by chunk

Reply:
BeautifulSoup is perfect for that